settings.gradle

```diff
@@ -186,3 +186,7 @@ include 'data-prepper-plugins:aws-lambda'
 include 'data-prepper-plugin-schema'
 include 'data-prepper-plugins:kinesis-source'
 include 'data-prepper-plugins:opensearch-api-source'
+include 'data-prepper-plugins:saas-source-plugins'
+include 'data-prepper-plugins:saas-source-plugins:saas-crawler'
```

Comment:
I wonder if `saas-crawler` is the right name here. What guides the use of SaaS here? It seems more like this is a crawler capability and not necessarily related to a SaaS product per-se.

Reply:
based on the discussion, renamed it to `source_crawler`